schemas/vscs.json

```diff
@@ -463,10 +463,28 @@
                 "type": "boolean",
                 "default": true,
                 "advanced": true,
-                "sectionEnd": "VSC Options",
                 "title": "Harden VSC",
                 "description": "Enables hardening configuration on VSC",
                 "propertyOrder": 520
+            },
+            "apply_vsc_config": {
```

Comment:
We don't need the boolean. If the paths are defined, we apply them. No need to add yet another property.

Reply:
Made the change!